src/containers/offer-page/create-or-edit-offer/CreateOrEditOffer.tsx

```diff
@@ -61,22 +59,28 @@ const CreateOrEditOffer: FC<CreateOrUpdateOfferProps> = ({
     ? OfferActionsEnum.Edit
     : OfferActionsEnum.Create
 
-  const onResponseError = (error?: ErrorResponse) => {
-    dispatch(
-      openAlert({
-        severity: snackbarVariants.error,
-        message: getErrorKey(error)
-      })
-    )
-  }
   const onResponse = (response: Offer | null) => {
-    dispatch(
-      openAlert({
-        severity: snackbarVariants.success,
-        message: `offerPage.${offerAction}.successMessage`
-      })
-    )
+    if (hash == '#offer') {
+      dispatch(
+        openAlert({
+          severity: snackbarVariants.success,
+          message: `offerPage.createOffer.extendedSuccessMessage.${userRole}`,
+          duration: 10000,
+          isExtended: true,
+          route: authRoutes.myOffers.path
+        })
+      )
+    } else {
+      dispatch(
+        openAlert({
+          severity: snackbarVariants.success,
+          message: `offerPage.${offerAction}.successMessage`
+        })
+      )
+    }
```

Comment:
Try to replace this duplication of `dispatch(openAlert(...))` with ternary operator. You will have single `dispatch` and `openAlert` calls, and conditional arguments for `openAlert`

Reply:
thanks for your comments, I've fixed them all. can you re-review, especially this one? whether I wrote the conditional arguments correctly or is there a way to optimize it more